scripts/release.ts

```diff
@@ -15,6 +20,8 @@ release({
   },
   toTag: (pkg, version) => `${pkg}@${version}`,
   logChangelog: async (pkgName) => {
+    if (pkgName === 'plugin-rsc') return
+
```

Comment:
This change will be removed later on, right? (and the one below)

Reply:
I'm not sure yet if I switch to `## Unreleased` convention. For now, I'll let this skip and use my convention.

Reply:
I'm fine with that. Could we have a small comment in `generateChangelog` method so that it's easier to find how the changelog is generated?

Reply:
I added a comment. Probably the release flow will be to create a PR to just update `CHANGELOG.md`, then merge it and then run `pnpm release` on main.